CRM/Civicase/Upgrader.php

```diff
@@ -554,7 +556,7 @@ public function enqueuePendingRevisions(CRM_Queue_Queue $queue) {
    *   The queue requires that true is returned on successful upgrade, but we
    *   use exceptions to indicate an error instead.
    */
-  public static function runStepUpgrade(CRM_Queue_TaskContext $context, $step) {
+  public static function runStepUpgrade(CRM_Queue_TaskContext $context, object $step) {
```

Comment:
Lets's avoid type hinting this for now. Dug up this PR where an error occurred in sites running <PHP 7.2: https://github.com/compucorp/uk.co.compucorp.civicase/pull/258.
It might cause an Issue on BASW.

Reply:
Sure, using `--no-verify`.